build/mulesoft_traceability_agent.yml

```diff
@@ -63,6 +63,12 @@ mulesoft_traceability_agent:
     level: ${LOG_LEVEL:info}
     format: ${LOG_FORMAT:json}
     output: ${LOG_OUTPUT:stdout}
+    metricfile:
+      enabled: ${LOG_METRICFILE_ENABLED:true}
+      name: ${LOG_METRICFILE_NAME:metrics.log}
+      keepfiles: ${LOG_METRICFILE_KEEPFILES:7}
+      rotateeverybytes: ${LOG_METRICFILE_ROTATEEVERYBYTES:10485760}
+      cleanbackups: ${LOG_METRICFILE_CLEANBACKUPS:0}
```

Comment:
thanks for this

Reply:
you're welcome.. but you made this change. I just added the extra commits from main into this 😄